ui/raidboss/data/06-ew/ultimate/dragonsongs_reprise_ultimate.txt

```diff
@@ -179,12 +187,235 @@ hideall "--sync--"
+3606.3 "Hot Wing" sync / 1[56]:[^:]*:Nidhogg:6D2C:/
+3607.2 "Spreading Flames and Entangled Flames" sync / 1[56]:[^:]*:Nidhogg:742B:/
+3611.1 "Mortal Vow" sync / 1[56]:[^:]*:Nidhogg:6D31:/
+3623.4 "Akh Afah" sync / 1[56]:[^:]*:Hraesvelgr:6D42:/
+3635.4 "Hallowed Wings" sync / 1[56]:[^:]*:Hraesvelgr:6D26:/
+3635.4 "Hot Tail" #sync / 1[56]:[^:]*:Nidhogg:6D2D:/
+3636.2 "Hallowed Plume and Hallowed Wings" sync / 1[56]:[^:]*:Hraesvelgr:6D29:/
+3636.4 "Hot Tail" #sync / 1[56]:[^:]*:Nidhogg:6D2E:/
+3645.1 "Mortal Vow" sync / 1[56]:[^:]*:Nidhogg:6D31:/
+3655.1 "Great Wyrmsbreath and Dread Wyrmsbreath" sync / 1[56]:[^:]*:Hraesvelgr:6D35:/
+3655.8 "Swirling Blizzard" sync / 1[56]:[^:]*:Hraesvelgr:6D38:/
+3655.9 "Ice Breath" #sync / 1[56]:[^:]*:Hraesvelgr:6D37:/
+3655.9 "Holy Orb" #sync / 1[56]:[^:]*:Hraesvelgr:6D3A:/
+3655.9 "Flame Breath" #sync / 1[56]:[^:]*:Nidhogg:6D36:/
+3655.9 "Dark Orb" #sync / 1[56]:[^:]*:Nidhogg:6D39:/
+3668.4 "Cauterize" sync / 1[56]:[^:]*:Hraesvelgr:6D3F:/
+3674.5 "--sync--" sync / 1[56]:[^:]*:Hraesvelgr:6D40:/
+3675.4 "Touchdown" sync / 1[56]:[^:]*:(Hraesvelgr|Nidhogg):70E7:/
+3679.1 "Mortal Vow" sync / 1[56]:[^:]*:Nidhogg:6D31:/
+3701.8 "Revenge of the Horde (enrage)" sync / 1[56]:[^:]*:(Hraesvelgr|Nidhogg):6D21:/
+
+3711.8 "Resentment?"
+3718.8 "Shockwave?" #sync / 1[56]:[^:]*:(King Thordan|Nidhogg):71E4:/
+3744.6 "Alternate End?" #sync / 1[56]:[^:]*:Dragon-king Thordan:7438:/
+
+
+# Phase 6.5: Eyes v2
+# Note: Shockwave could come from King Thordan or Nidhogg
+3809.5 "--sync--" sync / 1[56]:[^:]*:Right Eye:717A:/ window 200,0
+3809.5 "Resentment"
+3826.5 "Shockwave?" #sync / 1[56]:[^:]*:(King Thordan|Nidhogg):71E4:/
+3828.3 "--sync--" #sync / 1[56]:[^:]*:Left Eye:63F3:/
+3828.5 "--sync--" #sync / 1[56]:[^:]*:Right Eye:63F3:/
+3842.3 "Alternate End?" #sync / 1[56]:[^:]*:Dragon-king Thordan:7438:/
+
+# Phase 7: Dragon-king Thordan
+# Note: Nidhogg seems to get named for the first Chariot
+# Note: King Thordon can appear as one of the Exaflare casters in the first set
+# Note: Many of these spells are not named in garland db, so here is a name mapping:
+# Exaflare's Edge Spells:
+#   6D9B = Cast
+#   69DC = First Exaflare
+#   6D9D = Subsequent Exaflares
+# Akh Morn's Edge Spells:
+#  69D3 = Cast
+#  730C = 3 player tower (First hit)
+#  730D = 3 player tower (First hit)
+#  730E = 2 player tower (First hit)
+#  6D96 = 3 player tower (after first hit)
+#  6D97 = 2 player tower (after first hit)
+#  6D94 = Cast that checks if player is in towers?
+#  6D95 = Last cast in the Akh Morn in the sets in place of 6D94
+#  ???? = Raidwide spellid(s) if miss a tower
+# Trinity Spells:
+#  6D9E = Cast
+#  6D9F = Highest Emnity
+#  6DA0 = Second Highest Emnity
+#  6DA1 = Nearest Player
+# Gigaflare Spells:
+#  6D99 = Cast
+#  6D9A = First Proximity AoE
+#  6DD2 = Second Proximity AoE
+#  6DD3 = Third Proximity AoE
+# 6D91 = Fire Swords (Out/Chariot)
+# 6D92 = Ice Swords (In/Dynamo)
+# TODO: Get the proper names of 69D1 and 69D2?
```

Comment:
6D91 = https://xivapi.com/Action/28049?pretty=true = _rsv_28049_-1_1_C0_0Action

From https://discord.com/channels/551474815727304704/968569969103216651/970403963612561408, this is `Flames of Ascalon`.  So, 6D92 is `Ice of Ascalon`.

Reply:
Also, if you haven't seen this, copied this from elsewhere, the in/out here is based on Unknown_808 statusloop vfx effects, like DRS sword animations.  12A = fire, 12B = ice.

```
26|2022-05-01T07:14:55.0733962+02:00|808|Unknown_808|9999.00|E0000000||4000C932|Dragon-king Thordan|12A|12178508|
26|2022-05-01T07:15:40.6934917+02:00|808|Unknown_808|9999.00|E0000000||4000C932|Dragon-king Thordan|12B|12178508|
```

Do you know if these only happen once, and so would be a good trigger for the in/out call? I don't have any logs past what's been posted publicly in ffxiv act discord, and have seen past eyes yet myself.

Reply:
The words I retreived from carefully viewing VODs that have fflog uploaded. Typically the kill VODs don't have the player getting hit by an avoidable mechanic so I hadn't found the names of the avoidable spells haha.

I think this would be a good trigger for the in/out call, but FFlogs seems to be only catching Haurchefant with 808 and one time at that.

Reply:
Ah, ok, so you don't have a network log, just fflogs.  Got it.